scaleway/table_scaleway_account_project.go

```diff
@@ -0,0 +1,133 @@
+package scaleway
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/scaleway/scaleway-sdk-go/api/account/v3"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableScalewayAccountProject(_ context.Context) *plugin.Table {
```

Comment:
Can you please share the results from `select * from scaleway_account_project`?

Reply:
i have add a result in description.